### PR TITLE
Community Example: Go-Jetpack

### PR DESCRIPTION
**Pull request by bcamp1**
# Go-Jetpack
Hey, this is a community example I wrote called go-jetpack, where you are a blue gopher exploring the skies with a jet-pack. It ties in all of the fundamental components of Pixel. Hope you enjoy it!

### CODE REVIEW

**Comment by mewmew** [comment]
Looks great! Consider adding a readme with a screenshot to the `go-jetpack` directory.

screenshot_go-jetpack

Also, I know you deleted one of the `.DS_Store` files with commit https://github.com/faiface/pixel/pull/103/commits/04e61b90890cbe7a6d4ff2d9b410e84462b74967. There are three more to remove, at:

[.DS_Store](https://github.com/faiface/pixel/pull/103/files#diff-edb274d8c594429b8ccc684d79d771f6)
[examples/.DS_Store](https://github.com/faiface/pixel/pull/103/files#diff-6f7784d0fe5806b3d38790a96f328bf4)
[examples/community/.DS_Store](https://github.com/faiface/pixel/pull/103/files#diff-5718992adda0896f66274b31111330f6)

**Comment by bcamp1** [comment]
Thanks. Should be able to make those changes later today

**Comment by arxdsilva** [comment]
Hey guys, just as a comment I’d prefer if @bcamp1 rewrites the commits, so for future reference his work can be found easier. 

You can squash them all and rewrite as it is shown here (https://gist.github.com/landsman/d9d2b5fd03b0a3fff0c26603f3360987)

File: examples/community/go-jetpack/jetpack.go [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+		Bounds: pixel.R(0, 0, 1024, 768),
+		VSync:  true,
+	}
+	win, err := pixelgl.NewWindow(cfg)
+	if err != nil {
+		panic(err)
+	}
+
+	// Importantr variables
+	var jetX, jetY, velX, velY, radians float64 = 0, 0, 0, 0, 0
```
Comment (arxdsilva) [inline examples/community/go-jetpack/jetpack.go]: you don’t need to assign these variables to zero, it’s default value is zero (https://play.golang.org/p/qYENMZMRA3S)

**Comment by faiface** [comment]
@arxdsilva Yeah, I agree. All these commits don't have any meaning on their own. Would you be so kind, @bcamp1? Of course, you can do this after you're done with the changes (i.e. when you add a README with a screenshot, as well).

**Comment by bcamp1** [comment]
Sure! Ill squash the commits.

**Comment by bcamp1** [comment]
All done. I've squashed the past 12 commits into "added go-jetpack" and then made an additional commit to change `x, y, ... float64 = 0, 0, 0, 0` into `x, y, ... float64`

**Comment by faiface** [comment]
Okay, I guess all's fine now.

**Comment by faiface** [comment]
Thanks a lot!

**Pull request merged**